### ISSUE TITLE
Whether cogview4 can use Chinese as caption when training lora

### ISSUE THREAD

**Issue by huan085128** [a-r-r-o-w/finetrainers#334]
When I trained lora using Chinese as caption based on my own data set, I got very high losses, very poor validation results, and the model learned very little. Does anyone know why?

The training parameters are as follows:

```bash
training_cmd=(
  --training_type "lora"
  --seed 42
  --batch_size 1
  --train_steps 5000
  --rank 32
  --lora_alpha 32
  --target_modules "transformer_blocks.*(to_q|to_k|to_v|to_out.0)"
  --gradient_accumulation_steps 1
  --gradient_checkpointing
  --checkpointing_steps 1000
  --checkpointing_limit 2
  # --resume_from_checkpoint 3000
  --enable_slicing
  --enable_tiling
)

# Optimizer arguments
optimizer_cmd=(
  --optimizer "adamw"
  --lr 3e-5
  --lr_scheduler "constant_with_warmup"
  --lr_warmup_steps 1000
  --lr_num_cycles 1
  --beta1 0.9
  --beta2 0.99
  --weight_decay 1e-4
  --epsilon 1e-8
  --max_grad_norm 1.0
)
```

```bash
wandb: Run history:
wandb:       global_avg_loss █▃▄▃▄▄▆▄▇▄▃▅▅▆▄▇▆█▅▅▃▃▆▄▃▃▁▄▆▅▂▄▃▃▄▂▄▇▅▃
wandb:       global_max_loss ▄▄▄▅▄▂▄▄▄▅▃▂▂▁▃▂▅▁▃▆▂▂▅▅▄▃▆▂▅▄▅▂█▅▄▃▄▂▂▅
wandb:             grad_norm ▇▁▃▂▂▃▂▇▃▂▁▄▂▂▅▂▃▁▂▃▁▁▁▄▂▂▁▇▁█▁▁▄▁▂▂▇▃▃▃
wandb: observed_data_samples ▁▁▁▂▂▂▂▂▂▂▃▄▄▄▄▄▄▄▄▄▄▄▅▅▆▆▆▆▆▆▇▇▇▇▇▇████
wandb:   observed_num_tokens ▁▁▁▂▂▂▂▂▂▂▃▃▃▃▃▃▄▅▅▅▆▆▆▆▆▆▇▇▇▇▇▇▇▇▇▇████
wandb: 
wandb: Run summary:
wandb:       global_avg_loss 1.02946
wandb:       global_max_loss 1.13543
wandb:             grad_norm 0.09618
wandb: observed_data_samples 10000
wandb:   observed_num_tokens 15040000
```

**Comment by a-r-r-o-w** [comment]
I believe this is expected with both english/chinese captions. The expected loss for CogView4 LoRA training is between 0.8-1.5 from a few experiments I tried. I cross-checked this with @zRzRzRzRzRzRzR (he works closely with the CogView4 training team) and it seems consistent with what he observed as well. You will see that the model still learns concepts even though the loss is higher (also, at a small scale, the loss is not really representative of anything tbh)
